spec/install/typings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "protractor-install",
+  "dependencies": {},
+  "globalDependencies": {
+    "jasmine": "registry:dt/jasmine#2.2.0+20160621224255",
+    "node": "registry:dt/node#6.0.0+20160802155038",
+    "rimraf": "registry:dt/rimraf#0.0.0+20160317120654"
```

Comment:
Yes...will remove this. Originally test.js was in typescript and I was using rimraf in that.